### DISCUSSION TITLE
scala: error: not found: object play

### DISCUSSION THREAD

**Discussion by jharitesh108** [community/community#26501] [Actions]
<p>I am using default <a href="https://github.com/actions/starter-workflows/blob/master/ci/scala.yml" rel="nofollow noopener">https://github.com/actions/starter-workflows/blob/master/ci/scala.yml</a> for project based on playframework. But CI is failing with error. It is not able to find play while previous step of sbt installation went well.</p>
<p>Error:</p>
<pre><code>[info] Updated file /home/runner/work/bento/bento/project/build.properties: set sbt.version to 1.3.8
[info] Loading project definition from /home/runner/work/bento/bento/project
/home/runner/work/bento/bento/build.sbt:1: error: not found: object play
import play.sbt.PlayImport
       ^
/home/runner/work/bento/bento/build.sbt:15: error: not found: value PlayScala
lazy val root = (project in file(".")).enablePlugins(PlayScala)
                                                     ^
/home/runner/work/bento/bento/build.sbt:18: error: not found: value ws
        ws,
        ^
/home/runner/work/bento/bento/build.sbt:19: error: not found: value guice
        guice,
        ^
[error] sbt.compiler.EvalException: Type error in expression
[error] Use 'last' for the full log.
Project loading failed: (r)etry, (q)uit, (l)ast, or (i)gnore?
##[error]Process completed with exit code 1.
</code></pre>
<p>scala.yml:</p>
<pre><code>name: Scala CI

on:
  push:
    branches: [master]
  pull_request:
    branches: [master]

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2
    - name: Set up JDK 1.8
      uses: actions/setup-java@v1
      with:
        java-version: 1.8
    - name: Run tests
      run: sbt clean test
</code></pre>

**Comment by Yanjingzhu** [comment]
<p>Hi <a class="mention" href="/jharitesh108">@jharitesh108</a> , </p>
<p>I tried to use the same scala.yml to build a <a href="https://github.com/playframework/play-scala-starter-example" rel="nofollow noopener">playframework sample project</a>, the workflow run succeeded . There is my workflow run : <a href="https://github.com/zhuorg/play-scala-starter-example/runs/591222539?check_suite_focus=true" rel="nofollow noopener">https://github.com/zhuorg/play-scala-starter-example/runs/591222539?check_suite_focus=true</a></p>
<p>Could you please compare your project with this sample project? Such as the project/plugins.sbt file  ,build.sbt file?</p>
<p>Or would you mind sharing your project here? </p>

**Comment by jharitesh108** [comment]
<p>Thanks for ur help. I also created just one starter project and enable github action. It worked. <a href="https://github.com/smarterhealth/play-starter/runs/591504958?check_suite_focus=true" rel="nofollow noopener">https://github.com/smarterhealth/play-starter/runs/591504958?check_suite_focus=true</a> I will compare this with my problematic repo may be some config is not set as per github action expectation. Local build is running without any issue. I will post here later when I get reason. </p>
<p>Thanks again.</p>

**Comment by jharitesh108** [comment, marked as answer]
<p>I found the reason. In .gitignore file, I added "</p>
<pre><code>project   
</code></pre>
<p> instead of </p>
<pre><code>/.project
</code></pre>
<p>that’s why in local it was ok but on github action’s shell build was failing. A clear error about missing plugins.sbt file would have helped. </p>

**Comment by Yanjingzhu** [comment]
<p>Wow, I am so glad that you have found the root cause.</p>
<p>For the error message improvement, in my opinion , it would be better if you could share your idea with sbt command team . Thank you for your understanding. </p>
